src/pages/Login/Login.js

```diff
@@ -3,22 +3,67 @@ import { withRouter, Link } from 'react-router-dom';
 import './Login.scss';
 
 class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      acccount: '',
+      password: '',
+    };
+  }
+
+  handleLoginInput = e => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  goToMain = e => {
+    e.preventDefault();
+    fetch('http://10.58.2.84:8000/users/singin', {
+      method: 'POST',
+      body: JSON.stringify({
+        account: this.state.account,
+        password: this.state.password,
+      }),
+    })
```

Comment:
```suggestion
    const {account, password} = this.state;
    fetch('http://10.58.2.84:8000/users/singin', {
      method: 'POST',
      body: JSON.stringify({account, password}),
    })
```
- 객체에 넣어줄 key-value가 동일할 경우, 단축속성명 이용할 수 있습니다.
- 구조분해할당해서 동일한 이름으로 변수에 값 담은 뒤에 단축속성명 이용한 코드입니다!

Reply:
수정후 셀프 머지합니다!!!!!
nav바의 경우  validation 수정 및 아이디/비밀번호 찾기 구현 후 사항으로 넘어가서 조금 나중에 구현해보겠습니다!!